itest/integration_test.go

```diff
@@ -98,6 +98,10 @@ func TestTaroDaemon(t *testing.T) {
 	err = lndHarness.SetUp(ht.t, "taro-itest", lndDefaultArgs)
 	require.NoError(ht.t, err)
 
+	// Start aperture service and attach to test harness.
+	apertureHarness := setupApertureHarness(ht.t)
```

Comment:
We should also add `aperture.SetupLoggers(h.logWriter, h.interceptor)` to the `setupLogging()` method of `harnessTest`, then you see the aperture logs as well.